docs/guides/team/resource_operation_permission_rules/chat_application_permissions.md

```
@@ -1,13 +1,14 @@
-sidebar_label: "Chat application permissions"
+sidebar_label: Chat Application Permissions
+title: Chat Application Permissions
 ---
 
-## Chat application permissions
+# Chat Application Permissions
 
 Chat application permissions control viewing, configuration, creation, deletion, generation, and retrieval operations for chat assistants.
 
-### Chat management
+### Chat Management
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)